test/sharness/t0130-multinode.sh

```diff
@@ -78,4 +78,10 @@ test_expect_success "set up tcp testbed" '
 
 run_basic_test
 
+test_expect_success "set up utp testbed" '
+	iptb init -n 5 -p 0 -f --bootstrap=none --utp
```

Comment:
well yeah, thats because this command just inits things... the connection and connection checks happen above in `startup_cluster`